c/le.c

```diff
@@ -255,7 +255,7 @@ void deleteRLETask(RLETask *task) {
 #endif
 
 #ifdef __ZOWE_OS_ZOS
-void initRLEEnvironment(RLEAnchor *anchor) {
+int initRLEEnvironment(RLEAnchor *anchor) {
```

Comment:
Changing the return type may potentially break someone's code. Plus returning an RC from recovery.c can be misleading. Given that `recoveryEstablishRouter` shouldn't be failing too often, I think we should leave this alone. In the future we can:
* Abort instead of printing a message since a failure in `recoveryEstablishRouter` isn't something your application would want to continue with
* If needed make another version which returns proper error information

@1000TurquoisePogs what do you think?